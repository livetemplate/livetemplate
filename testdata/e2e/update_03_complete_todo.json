{"6":"2","7":"0","8":"100%","9":[["r",""],["u","completed",{"0":"6","2":"#1","3":"Write documentation","5":"(Priority: Low)"}]],"10":"2023-01-01 10:45:00"}