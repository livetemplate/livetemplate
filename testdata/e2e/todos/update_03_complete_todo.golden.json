{
  "5": "1",
  "6": "1",
  "7": "50%",
  "8": {
    "0": {
      "d": [
        {
          "0": "",
          "1": "todo-3",
          "2": "",
          "3": "Write documentation",
          "4": "○",
          "5": " (Priority: Low)"
        },
        {
          "0": "completed",
          "1": "todo-1",
          "2": "",
          "3": "Learn Go templates",
          "4": "✓",
          "5": " (Priority: High)"
        }
      ],
      "s": [
        "\n                    \u003cdiv class=\"todo-item ",
        "\" data-key=\"",
        "\"\u003e\n                        \u003cstrong\u003e",
        ":\u003c/strong\u003e\n                        ",
        "\n                        ",
        "\n                        ",
        "\n                    \u003c/div\u003e\n                    "
      ]
    },
    "s": [
      "\n                \u003cdiv class=\"todo-list\"\u003e\n                    ",
      "\n                \u003c/div\u003e\n                "
    ]
  },
  "9": "2023-01-01 10:45:00"
}